internal/endpoints/metrics.go

```diff
@@ -15,11 +15,16 @@ var (
 		Help: "Number of requests to the payload tracker.",
 	}, []string{})
 
-	invalidRequests = pa.NewCounterVec(p.CounterOpts{
-		Name: "payload_tracker_invalid_requests",
+	apiInvalidRequests = pa.NewCounterVec(p.CounterOpts{
+		Name: "payload_tracker_api_invalid_requests",
 		Help: "Number of invalid requests to the payload tracker.",
```

Comment:
Can we be a bit more specific with the name and help? It kind of reads like these are invalid request tot he api, rather than it being the requestID specifically that's an issue